### PR TITLE
feat-12-lists

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Lists component.

### Preview:
1.Spaced Lists:
image

2.Stacked Lists:
image

### CODE REVIEW

File: docs/lists.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                <div class="component-showcase lists-spaced">
+                    <ul class="spaced-lists">
+                        <li>Item 1</li>
+                        <li>Item 2</li>
+                        <li>Item 3</li>
+                    </ul>
+                </div>
+
+                <pre><code class="code-snippet">
+<!-- add class="spaced-lists" in the <ul> tag-->
```
Comment (PuneetP16) [inline docs/lists.html:76]: remove the comment if no longer useful

File: stylesheets/components/lists/lists.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+@import "/main.css";
+
+.spaced-lists{
+    display: flex;
+    width: 100%;
+    justify-content: space-around;
+    gap: 1rem   
+}
```
Comment (PuneetP16) [inline stylesheets/components/lists/lists.css]: Semicolon missing on lines 7 and 18

**Review by PuneetP16 — APPROVED** [review]
Check for the comments. Looks good to merge.

File: stylesheets/components/lists/lists.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
+@import "/main.css";
+
+.spaced-lists{
+    display: flex;
+    width: 100%;
+    justify-content: space-around;
+    gap: 1rem
```
Comment (pankaj-prog) [inline stylesheets/components/lists/lists.css]: keep flex properties all together, its easy to read when you have multiple properties

**Review by pankaj-prog — APPROVED** [review]

File: stylesheets/components/lists/lists.css (hunk 3 above)
Comment (SinchanaMS) [reply stylesheets/components/lists/lists.css]: Thanks for pointing it out, Pankaj! I missed formatting this.

File: docs/lists.html (hunk 1 above)
Comment (SinchanaMS) [reply docs/lists.html:76]: This comment is within the code snippet, put intentionally.

**Comment by SinchanaMS** [comment]
Thank you for the review. Merging to temp-dev.

**Pull request merged**